models/engine/db_storage.py

```diff
@@ -88,3 +88,33 @@ def close(self):
             Remove private session attribute
         '''
         self.__session.close()
+
+    def get(self, cls, id):
+        '''
+            Retrieve an obj w/class name and id
+        '''
+        result = None
+        try:
+            objs = self.__session.query(models.classes[cls]).all()
+            for obj in objs:
+                if obj.id == id:
+                    result = obj
+        except:
+            pass
+        return result
+
+    def count(self, cls=None):
+        '''
+            Count number of objects in DBstorage
+        '''
+        cls_counter = 0
+
+        if cls != None:
+            objs = self.__session.query(models.classes[cls]).all()
+            cls_counter = len(objs)
+        else:
+            for k, v in models.classes.items():
+                if k != "BaseModel":
+                    objs = self.__session.query(v).all()
+                    cls_counter += len(objs)
```

Comment:
Good decision to use python builtin 'len' instead of a for loop to count objects.